### PR TITLE
rename crate and set version

### PR DESCRIPTION
**Pull request by buffalojoec**

### CODE REVIEW

File: program/Cargo.toml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 [package]
-name = "stake-program"
-version = "0.0.0"
+name = "paladin-stake-program"
+version = "0.0.1"
 edition = "2021"
 readme = "./README.md"
 license-file = "../LICENSE"
```
Comment (febo) [inline program/Cargo.toml:6]: If the plan is to publish this crate, then we need to remove the line below:
```
publish = false
```

File: clients/js/src/generated/index.ts [hunk 2]
```diff
@@ -9,6 +9,5 @@
 export * from './accounts';
 export * from './errors';
 export * from './instructions';
-export * from './pdas';
```
Comment (febo) [inline clients/js/src/generated/index.ts]: I can't comment on the `generate-clients.mjs` file, but we are missing a rename there. This is why the `pdas` folder is not being generated.

https://github.com/paladin-bladesmith/stake-program/blob/62b2dc6dce04b520e411d2e2fa82907eacdb0b76/scripts/generate-clients.mjs#L26

```
"[programNode]stake" -> "[programNode]paladinStakeProgram"

**Comment by febo** [comment]
We can also remove lines `15-20` from the Kinobi configuration since we are not renaming the program anymore.

https://github.com/paladin-bladesmith/stake-program/blob/62b2dc6dce04b520e411d2e2fa82907eacdb0b76/scripts/generate-clients.mjs#L15-L20

**Comment by buffalojoec** [comment]
Thanks @febo, all set!

**Review by febo — APPROVED** [review]
Looks great!

**Pull request merged**